SPECS/kernel/config

```diff
@@ -280,7 +288,7 @@ CONFIG_SLUB_DEBUG=y
 # CONFIG_SLAB is not set
 CONFIG_SLUB=y
 # CONFIG_SLOB is not set
-# CONFIG_SLAB_MERGE_DEFAULT is not set
+CONFIG_SLAB_MERGE_DEFAULT=y
```

Comment:
reduces kernel memory fragmentation but leaves attack vector higher, leave off for now

https://cateee.net/lkddb/web-lkddb/SLAB_MERGE_DEFAULT.html